addons/html_editor/static/src/fields/html_viewer.js

```diff
@@ -221,31 +218,26 @@ export class HtmlViewer extends Component {
             env,
             props,
         });
-        const app = new App(Component, {
-            test: dev,
-            env,
-            translateFn,
-            getTemplate: getRawTemplate,
+        const app = this.__owl__.app;
+        const root = app.createRoot(Component, {
```

Comment:
Not a big deal but could write `this.__owl__.app.createRoot(...)` in one line? Otherwise lgtm, maybe the comment about the mount patch could be rephrased to use "root" instead of "app" too